CSharp/BatchExplorer/Converters/InvertedBooleanConverter.cs

```diff
@@ -12,18 +12,28 @@ namespace Microsoft.Azure.BatchExplorer.Converters
     public class InvertedBooleanConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
-        {
-            if (targetType != typeof (bool))
-            {
-                throw new InvalidOperationException("The target must be a bool.");
-            }
-
-            return !(bool)value;
-        }
+        {
+            return _Convert(value, targetType);
+        }       
 
         public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
-        {
-            throw new NotImplementedException();
+        {
+            return _Convert(value, targetType);
+        }
+
+        private static object _Convert(object value, Type targetType)

```

Comment:
The underscore prefix for method names is unorthodox. Why not just Convert, or ConvertImpl or ConvertCore if you want to emphasise that this is a method delegated to by the public ones?